src/site/_includes/summary.njk

```diff
@@ -7,6 +7,7 @@
 <div class="template-summary">
   <div class="box">
     <div class="left">
+      <h3><a class="cta-title" href="{{ item.url }}">{{ item.data.title }}</a></h3>
```

Comment:
The h3 is still visible on desktop, and its margin shifts the images down - the whole h3 should be hidden when not visible.

Reply:
Ohh, good catch! Thanks!